api-tests/src/clients/cri-stub-client.ts

```diff
@@ -5,6 +5,25 @@ import {
 } from "../types/cri-stub.js";
 import config from "../config/config.js";
 
+const CREDENTIAL_ISSUER_SUBDOMAINS: Record<string, string> = {
+  ticf: "ticf",
+  kbv: "experian-kbv-cri",
+  dcmaw: "dcmaw-cri",
+  address: "address-cri",
+  fraud: "fraud-cri",
+  ukPassport: "passport-cri",
+  drivingLicence: "driving-license-cri",
```

Comment:
Is this really `driving-license-cri` 😢   (as opposed to `driving-licence`)

Reply:
yeah :')